app/javascript/chat/ActiveChatChannel/IntroductionMessages/OpenIntroMessage.js

```diff
@@ -0,0 +1,22 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+const OpenInroMessage = ({ activeChannel }) => {
+  return (
+    <div className="chatmessage" style={{ color: 'grey' }}>
+      <div className="chatmessage__body">
+        You have joined {activeChannel.channel_name}! All interactions{' '}
```

Comment:
refactor (non-blocking): Consider using a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead.

```suggestion
        {`You have joined ${activeChannel.channel_name}! All interactions `}
```